src/renderer/git/log.js

```diff
@@ -1,31 +1,23 @@
 import nodegit from "nodegit";
 
-const log = async path => {
+const log = async (path, count = 50) => {
```

Comment:
I put it 50 items by default. I think that is a good idea having the parameter just in case.

Reply:
Putting a hard-coded value `50` is not the best way to overcome this issue.

We need to think of better solution.